CMakeLists.txt

```diff
@@ -288,6 +287,5 @@ endif()
 if(EXISTS ${USB_IDS_GZ_PATH} AND NOT EXISTS ${USB_IDS_INSTALL_PATH})
     message(STATUS "Extracting USB ID database from ${USB_IDS_GZ_PATH}...")
     file(MAKE_DIRECTORY ${CMAKE_INSTALL_DATAROOTDIR})
-    file(COPY ${USB_IDS_GZ_PATH} DESTINATION ${CMAKE_INSTALL_DATAROOTDIR})
-    execute_process(COMMAND gzip -d -o ${USB_IDS_INSTALL_PATH} ${CMAKE_INSTALL_DATAROOTDIR}/${USB_IDS_FILE}.gz)
+    execute_process(COMMAND gzip -d -c ${USB_IDS_GZ_PATH} > ${USB_IDS_INSTALL_PATH})
```

Comment:
[The documentation for execute_process](https://cmake.org/cmake/help/latest/command/execute_process.html) says:

> CMake executes the child process using operating system APIs directly. All arguments are passed VERBATIM to the child process. No intermediate shell is used, so **shell operators such as > are treated as normal arguments**. (Use the INPUT_*, OUTPUT_*, and ERROR_* options to redirect stdin, stdout, and stderr.)

`OUTPUT_FILE` seems to be the right argument here.

Reply:
Was just about to comment the same :)
As is, this patch just prints the contents of the archives to my terminal. The following worked for me:

`execute_process(COMMAND gzip -d -c "${PCI_IDS_GZ_PATH}" OUTPUT_FILE "${PCI_IDS_INSTALL_PATH}")`
`execute_process(COMMAND gzip -d -c "${USB_IDS_GZ_PATH}" OUTPUT_FILE "${USB_IDS_INSTALL_PATH}")`

Reply:
Alrighty, should be squared away now. Thanks for the review (and cluestick that I should have checked the docs first)!